test/Elastic.Apm.SqlClient.Tests/Elastic.Apm.SqlClient.Tests.csproj

```diff
@@ -1,15 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
     <RootNamespace>Elastic.Apm.SqlClient.Tests</RootNamespace>
     <AssemblyName>Elastic.Apm.SqlClient.Tests</AssemblyName>
+    <TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="5.6.0" />
     <PackageReference Include="Microsoft.Data.SqlClient" Version="1.0.19269.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
+    <PackageReference Include="StrongNamer" Version="0.2.5" />
```

Comment:
Why do we need this? 🤔

Reply:
It's needed to solve the limitation of strong-named assembly in .Net Framework: signed assembly cannot load unsigned assemblies. .Net Core doesn't have such limitation and only raise warning during the build. This is why there wasn't a need it before. With adding `net461` target it was an issue